testutil/options/options.go

```diff
@@ -40,24 +40,35 @@ func SleepCreateOpts(num int) *options.Create {
 // ValidMongoDBDownloadOptions returns valid options for downloading a MongoDB
 // archive file.
 func ValidMongoDBDownloadOptions() options.MongoDBDownload {
-	target := runtime.GOOS
-	if target == "darwin" {
-		target = "osx"
+	return options.MongoDBDownload{
+		BuildOpts: ValidMongoDBBuildOptions(),
+		Releases:  []string{"7.0-current"},
```

Comment:
should we maybe avoid hardcoding `"7.0-current"`?

Reply:
I guess we could move it to a project var, but I'm not sure that's a lot better since it would invisibly make the test run with a different MongoDB version.

So far, this is the only time I've ever had to do this in the many years since it was written, but we can reconsider or possibly remove/rework this test to not need `mongod` if it happens more.